<launch>

  <node pkg="fixed_hk_onvif" type="raw_temp_node" name="raw_temp_node" respawn="false" output="screen">
      <param name="camera_id" type="int" value="1"/>
      <param name="cali_offset_h" type="int" value="90"/>
      <param name="cali_offset_v" type="int" value="0"/>
      <param name="camera_file" value="$(find fixed_hk_camera)/launch/66color_map0_-10.yaml"/>
  </node>
  
</launch>


